iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/DanglingLineData.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.util;
+
+import org.apache.commons.math3.complex.Complex;
+import com.powsybl.iidm.network.DanglingLine;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class DanglingLineData {
+
+    private final String id;
+
+    private final double r;
+    private final double x;
+    private final double g1;
+    private final double g2;
+    private final double b1;
+    private final double b2;
+
+    private final double u1;
+    private final double theta1;
+    private final double p0;
+    private final double q0;
+
+    double boundaryBusU;
+    double boundaryBusTheta;
+
+    public DanglingLineData(DanglingLine danglingLine) {
+
+        id = danglingLine.getId();
+        r = danglingLine.getR();
+        x = danglingLine.getX();
+        g1 = danglingLine.getG();
+        b1 = danglingLine.getB();
+        g2 = 0.0;
```

Comment:
Do we miss the parameter ````twtSplitShuntAdmittance```` here ? It is a parameter for the loadflow but it could be used here.

Reply:
The `twtSplitShuntAdmittance` parameter is only for transformers. The shunt admittance is always split in lines, but we are not sure about dangling lines. We are checking if we should split or not.

Reply:
Why do you say that the shunt admittance is always split for lines and dangling lines ? In IIDM, you can have what you want as B1, G1, B2 and G2 and it does not mean for me that it is split in 2 equal parts. So where do you have seen this in the code ? If you are right, I am not sure to understand why we leave the possibility to set B1, G1, B2 and G2. In a dangling line, as you have only B and G, you have the right to split it or not when running the powerflow. In the open loadflow, we split it for example.

Reply:
When I said that the shunt admittance is always split for lines I was referring in the cgmes conversion.  In CGMES model the total shunt admittance is received and is split in the conversion process. I prefer the line model with G1, B1, G2, B2 attributes as it is more generic and allows a "clean convert process" from standard tools, like `Psse`, that use in line model the same four attributes. I should prefer also to have the four attributes in the `danglingLine` model, to be coherent with the `line` model and to eliminate the possibility of splitting when running the load flow.  There is only one `real model` so i prefer to have an` iidm model` that allows me to eliminate all these ambiguities in the convert process.

Reply:
Thanks for your answer. I think that for dangling line, we only have a B and a G because a dangling line is a part of a merged line, but I agree if we have a computation on a not merged network, it is wierd. 
As you PR is over IIDM (and not CGMES model), we have to take B1, G1, B2 and G2 for a line and for dangling line, I agree that it could be better to split it as you have done as we don't have the 4 attributes...